src/logger.js

```diff
@@ -5,7 +5,7 @@
  * @implements {config}
  */
 
-const { debug } = require("./config.js").getConfig();
+const config = require("./config.js").getConfig();
```

Comment:
```suggestion
const { debug } = require("./config.js").getConfig();
```

Leaving the `debug` value as a variable lets us import it the same way we do with all other `config` values.